### PR TITLE
add to support jumper EZpad_8_pro

### PR DESCRIPTION
**Pull request by 240038901**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,7 +58,8 @@ for copyright information.
 | jumper        | EZpad 5SE                       | no      | [firmware/jumper/EZpad_5SE](firmware/jumper/EZpad_5SE)         |
 | jumper        | EZpad 6                         | yes     | [firmware/jumper/EZpad_6](firmware/jumper/EZpad_6)             |
 | jumper        | EZpad 6(S) Pro                  | yes     | [firmware/jumper/EZpad_6_pro](firmware/jumper/EZpad_6_pro)     |
-| jumper        | EZpad 6(S) Pro b                | yes     | [firmware/jumper/EZpad_6_pro_b](firmware/jumper/EZpad_6_pro_b) |
+| jumper        | EZpad 6(S) Pro b                | yes     | [firmware/jumper/EZpad_6_pro_b](firmware/jumper/EZpad_6_pro_b)
```
Comment (onitake) [inline README.md:61]: You removed the | at the end of the line. Please fix.

**Review by onitake — CHANGES_REQUESTED** [review]
SileadTouch.sys, firmware.fw and silead_ts.fw are all empty.
Please add the the file contents.

File: README.md (hunk 1 above)
Comment (240038901) [reply README.md:61]: fixed

Comment (240038901) [reply README.md:61]: I have already uploaded the SileadTouch.sys, firmware.fw and silead_ts.fw, please check it again.

**Comment by onitake** [comment]
Looks good now, thank you very much!

Github still shows the files as 0 bytes, but that seems to be a Github bug. If I look into your branch, I can see that they are not empty.

**Pull request merged**